pkg/webhook/certs.go

```diff
@@ -0,0 +1,369 @@
+package webhook
+
+import (
+	"bytes"
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"math/big"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+	"time"
+)
+
+const (
+	caName     = "gatekeeper-ca"
+	namespace  = "gatekeeper-system"
+	service    = "gatekeeper-webhook-service"
+	certName   = "tls.crt"
+	keyName    = "tls.key"
+	caCertName = "ca.crt"
+	caKeyName  = "ca.key"
+)
+
+var crLog = logf.Log.WithName("cert-rotation")
+
+var (
+	secretKey = types.NamespacedName{
+		Namespace: namespace,
+		Name:      "gatekeeper-webhook-server-cert",
```

Comment:
Same here, is there a reason this secret name is changed? It was `gatekeeper-webhook-server-secret`

Reply:
Given that we already have breaking changes (webhook name, StatefulSet -> Service, etc.), we may as well make this conform to the new standards as well.

Also, the name of the certs within the secrets have changed, per #111, so this may be a good way to highlight that.